experimental/sdg-tracker/src/state/index.ts

```diff
@@ -48,6 +48,7 @@ const MenuImageIcon = styled.img`
   border-radius: 0.25rem;
 `;
 
+const REGION_PLACE_TYPES = ["UNGeoRegion", "GeoRegion", "ContinentalUnion"];
```

Comment:
To confirm, do we want `Continent` as part of this list? Not sure if the UNGeoRegions/GeoRegions include whole continents or if we have data for them.

Reply:
@n-h-diaz can you confirm these are the 3 regions we need?

Reply:
yes we do have data for 'Continent' too, but I was thinking of this as separate from regions (so yes those are the only three region types). not sure if we also want Continent, but maybe that's a question for UN